todo_app/templates/index.html

```diff
@@ -12,7 +12,23 @@ <h1 class="display-4">To-Do App</h1>
       <h2>Items</h2>
       <ul class="list-group mb-4">
         <!-- List the todo items here -->
+		{% for todolist in todolists %}
+			<li>{{todolist.id}}        {{todolist.title}}          {{todolist.status}}</li>
+		{% endfor %}
       </ul>
     </div>
   </div>
+  
+  
+  <form action="http://localhost:5000/addtodoitem" method="post" class="add-item">
```

Comment:
You should use a relative url, e.g. `action="/addtodoitem"` or `action="{{ url_for('addtodoitem') }}"`

Otherwise this will break if you try to deploy this app to a server.